packages/flutter_tools/lib/src/usage.dart

```diff
@@ -44,35 +44,43 @@ class Usage {
   Analytics _analytics;
 
   bool _printedUsage = false;
+  bool _suppressAnalytics = false;
```

Comment:
Yeah either way is fine. I just like to see these kinds of things be obviously "fail safe".